apps/docs/content/guides/auth/auth-email.mdx

```diff
@@ -126,7 +126,7 @@ Future<void> signInWithEmail() async {
 </TabPanel>
 <TabPanel id="swift" label="Swift">
 
-When your user signs in, call [signInWithPassword()](/docs/reference/swift/auth-signinwithpassword) with their email address and password:
+When your user signs in, call [signIn(email:password:)](/docs/reference/swift/auth-signinwithpassword) with their email address and password:
```

Comment:
In Swift param names is part of the method signature, and this is format used to reference it and is how a Swift developer expects to see method references.